mambaSharedFramework/Pantos-Generic Playlist Parsing/Pantos-Generic Tag Parsers/NoOpTagParser.swift

```diff
@@ -19,12 +19,12 @@
 
 import Foundation
 
-/// "No operation" Tag Parser. This is returned in cases when `RegisteredHLSTags` cannot determine a `HLSTagParser`. Usage of this class in production is a error.
-public class NoOpTagParser: HLSTagParser {
+/// "No operation" Tag Parser. This is returned in cases when `RegisteredTags` cannot determine a `TagParser`. Usage of this class in production is a error.
```

Comment:
The use of `RegisteredTags` and `TagParser` don't follow the general convention of replacing `HLS` with `Playlist`. In addition, line 23 below uses `PlaylistTagParser` vs. `TagParser`.

Reply:
Fixed